platform/src/main/java/org/conscrypt/ct/CTLogStoreImpl.java

```diff
@@ -220,10 +218,10 @@ public static CTLogInfo loadLog(InputStream input) throws InvalidLogFileExceptio
 
         PublicKey pubkey;
         try {
-            pubkey = OpenSSLKey.fromPublicKeyPemInputStream(new StringBufferInputStream(
+            pubkey = InternalUtil.readPublicKeyFromPem(new StringBufferInputStream(
```

Comment:
There is no `From` in the new method name you made: `InternalUtil.readPublicKeyPem`